content/blog/2022-02-28-using-iterative-terraform-provider-with-docker.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Using Iterative Terraform Provider with Docker
+date: 2022-02-28
+description: >
+  In this tutorial you'll learn how you can move use an existing Docker image with 
+  Iterative Terraform Provider, as well as build your own Docker image.
+descriptionLong: >
+  In this tutorial you'll learn how you can move use an existing Docker image with 
+  Iterative Terraform Provider, as well as build your own Docker image.
```

Comment:
Should the short desc be shorter or the long one longer? Or both

Reply:
done